portfolio/src/main/webapp/script.js

```diff
@@ -90,15 +90,15 @@ function handleFirstTab(e) {
 
 /**
  * Loads guest book comments. 
- * TODO: standardize quotation marks; disallow blank comments
+ * TODO: disallow blank comments, deal with weird unicode error
```

Comment:
You can disallow blank comments pretty easily by just adding "required" inside the form bracket.